src/services/db/GitHubService.ts

```diff
@@ -507,15 +508,16 @@ export default class GitHubService {
   async getTree(
     sessionData: UserWithSiteSessionData,
     githubSessionData: GithubSessionData,
-    { isRecursive }: { isRecursive: any }
-  ) {
+    { isRecursive }: any,
```

Comment:
why is this typed as `any`? not a blocker as the prev revision already typed it as `any` too

Reply:
solely cuz RepoService typed it as any, can become stricter in the future